lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -145,7 +144,10 @@
                                 message = response.user_message;
                             }
                             view.showMessage(message, message);
-                        });
+                        })
+                    ViewUtils.disableElementWhileRunning($(".action-primary"), function() {
+                        return deferred.promise();
```

Comment:
The only place I found was here: https://github.com/edx/edx-platform/blob/d38da773ea2ff4b139e2c6b6d61e02681584fcab/common/static/common/js/spec/components/view_utils_spec.js#L15

I ended up removing the `resolve()` call entirely, that was throwing errors in js tests. That still seems to work.

So yeah, you're right - the anonymous method here returns a promise, `disableElementWhileRunning` does the resolve internally.